porcupine/plugins/run/no_terminal.py

```diff
@@ -477,3 +477,16 @@ def run_command(command: str, cwd: Path) -> None:
     assert runner is not None
     get_vertical_panedwindow().paneconfigure(runner.textwidget, hide=False)
     runner.run_command(cwd, command)
+    # Use previous height as long as it's not less than two lines high,
+    # otherwise set height to two lines high.
+    current_height = textutils.textwidget_size(runner.textwidget)[1]
+    padding = textutils.get_padding(runner.textwidget)[0]
+    linespace = tkinter.font.Font(font="TkFixedFont").metrics("linespace")
+    linespace += padding
+    # Show two lines of output
+    linespace *= 2
```

Comment:
This calculation does the right thing, but it's misleading.

In pseudo code, it calculates:

```
height = (linespace + padding) * 2
```

The correct calculation would be:

```
height = 2*linespace + 2*padding
```
where `2` in front of linespace means we make it two lines tall, and `2` in front of padding means we take into account the padding on top and the padding on bottom. Mathematically these are the same, but I would prefer a calculation that looks more like `2 * linespace + 2 * padding`.